src/Styles/Onboarding/useModuleCardStyles.js

```diff
@@ -25,17 +25,24 @@ export const useModuleCardStyles = makeStyles(theme => ({
         color: "#f6f6f6",
         borderRadius: "20px",
         textTransform: "uppercase",
-        [theme.breakpoints.down('sm')]: {
+        [theme.breakpoints.down("sm")]: {
             display: "none",
         },
     },
+    // added a new style to change the style of specifically the text
+    cardHeadingText: {
+        fontSize: "22px",
```

Comment:
Should we maybe use `em` or `rem` instead of `px`? That way, our font size can scale with the user's screen size and resolution.